projects/demo/src/modules/app/stackblitz/project-files/configs.md

```diff
@@ -21,7 +21,7 @@
             "polyfills": "src/polyfills.ts",
             "tsConfig": "src/tsconfig.app.json",
             "assets": ["src/favicon.ico", "src/assets"],
-            "styles": ["src/app/@stackblitz/styles/taiga-ui-stackblitz.less", "src/styles.less"]
+            "styles": ["src/styles.less"]
```

Comment:
To locally check that everything actually works, you should locally replace this line
https://github.com/Tinkoff/taiga-ui/blob/0acd7acab2752f078f1afc792eff782b59aa3993/projects/demo/src/modules/app/stackblitz/stackblitz-deps.constants.ts#L39

with 
```
'@ng-web-apis/common': `2.0.1`
```
(or just wait for the next release)

Read more: https://github.com/Tinkoff/taiga-ui/issues/3383#issuecomment-1375324393